collect_app/src/main/java/org/odk/collect/android/formlists/blankformlist/BlankFormListViewModel.kt

```diff
@@ -211,7 +211,7 @@ class BlankFormListViewModel(
         private const val SYNC_NOTIFICATION_CHANNEL = "form_updates"
         private const val SYNC_NOTIFICATION_CHANNEL_NAME = "Form updates"
 
-        private const val SYNC_NOTIFICATION_ID = 1
+        private const val SYNC_NOTIFICATION_ID = 3
```

Comment:
It seems like we've made an assumption in the code base that notifications are identified by their ID **and** their channel. In fact, they are only identified by ID, so this should be unique.

Reply:
Could you elaborate on this? Why 3? What was the issue here?